test/Spec.hs

```diff
@@ -1,4 +1,9 @@
 module Main (main) where
 
+import Config
+import Crocodealer.Config
+
 main :: IO ()
-main = putStrLn ("Test suite not yet implemented" :: String)
+main = do 
+    config <- loadConfig "example-config.toml"
+    testConfig config
```

Comment:
You should call `hspec` function explicitly. If you write top-level type signatures of the `testConfig` function, you will see that it looks like this:

```haskell
testConfig :: Config -> Spec
```

`Spec` is not the same thing as `IO`.  You need to call a function:

```haskell
hspec :: Spec -> IO ()
```
I hope this helps!
```suggestion
    hspec $ testConfig config
```

Reply:
crazy, this really helps :) Thank you!